doc/release-process.md

```diff
@@ -116,7 +116,7 @@ Ensure gitian-builder is up-to-date:
 
     pushd ./gitian-builder
     mkdir -p inputs
-    wget -O osslsigncode-2.0.tar.gz -P inputs https://github.com/mtrojnar/osslsigncode/archive/2.0.tar.gz
+    wget -O inputs/osslsigncode-2.0.tar.gz https://github.com/mtrojnar/osslsigncode/archive/2.0.tar.gz
```

Comment:
Just saying that with `-P inputs -O osslsigncode-2.0.tar.gz` the above `mkdir` could be removed - at least on my system. I prefer this way.

Reply:
May I get your wget version?  I'm running on bionic GNU wget 1.17.1 and for my wget, the `-P` gets ignored when there's `-O`
When I run `wget -P inputs -O osslsigncode-2.0.tar.gz https://github.com/mtrojnar/osslsigncode/archive/2.0.tar.gz`,
it does not place the output file osslsigncode-2.0.tar.gz in the inputs directory regardless of whether inputs directory exist or not

Reply:
I very much prefer making the directory explicitly as it is done now instead of relying on implicit wget behavior.